articles/vmware/vmw-faq.md

```diff
@@ -473,29 +463,27 @@ At the end of your trial, you have the option to transition to production or cea
 
 ## Protection
 
-### Which protection types can I purchase together?
-
-- Synchronous Protection & 14/28 Day Snapshot Protection
+### Can I create a clone of my environment?
 
-- Synchronous Protection & 14/28 Day Journaling Protection
+Yes, you can do this through the UKCloud Portal by right-clicking on a vApp and selecting 'Copy'. This operation can also be performed programmatically via the API.
 
-- Synchronous Protection, 14 Day Journaling Protection & 14 Day Snapshot Protection
+It's not possible to restrict the automated VM snapshot backup to specific files or directories. You can install your own backup service and use our Cloud Storage if you need more flexibility.
 
-- Synchronous Protection, 28 Day Journaling Protection & 28 Day Snapshot Protection
+## Can I still use Synchronous Protection for my VMs?
 
-- Synchronous Protection & 14 Day Journaling Protection & 28 Day Snapshot Protection
+Synchronous Protection is no longer available as protection option for UKCloud for VMware. We recommend using one of our other protection options: Journaling Protection or Snapshot Protection. We will continue to provide support to customers who previously added Synchronous Protection to their environment, although we would encourage considering using Journaling Protection instead.
```

Comment:
i would remove the last sentence ", although we would encourage considering using Journaling Protection instead."

Reply:
Done